.pre-commit-config.yaml

```diff
@@ -64,6 +56,12 @@ repos:
   - id: flake8
     additional_dependencies:
     - flake8-cognitive-complexity ~= 0.1.0
+    - flake8-docstrings
+    args:
+    - --docstring-convention=pep257
```

Comment:
Move this setting to `.flake8`.

Reply:
I noticed in the docs that the default is pep257. That's why I deleted it and didn't change `.flake8`.

Reply:
Marking this as resolved.